pulsar/resource_pulsar_topic.go

```diff
@@ -86,6 +86,23 @@ func resourcePulsarTopic() *schema.Resource {
 					},
 				},
 			},
+			"retention_policies": {
```

Comment:
Do we need to set the `Set` function for the retention_policies set here? like `Set: retentionPoliciesToHash`

Reply:
It is unnecessary, retention_policies only has an item.